hail_search/test_search.py

```diff
@@ -300,9 +300,9 @@ async def test_inheritance_filter(self):
             inheritance_mode=inheritance_mode, gene_counts={
                 'ENSG00000097046': {'total': 2, 'families': {'F000002_2': 2}},
                 'ENSG00000177000': {'total': 2, 'families': {'F000002_2': 2}},
-                'ENSG00000275023': {'total': 2, 'families': {'F000002_2': 2}},
-                'ENSG00000277258': {'total': 2, 'families': {'F000002_2': 2}},
-                'ENSG00000277972': {'total': 1, 'families': {'F000002_2': 1}},
+                'ENSG00000275023': {'total': 3, 'families': {'F000002_2': 3}},
```

Comment:
Behavior for gene counts changes for how it counts variants thats are included as part of multiple comp het pairs. While different than before, it is still accurate just with a different definition